src/libraries/System.Private.CoreLib/src/System/IO/PinnedBufferMemoryStream.cs

```diff
@@ -1,23 +1,14 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-/*============================================================
-**
-**
-**
-**
-**
-** Purpose: Pins a byte[], exposing it as an unmanaged memory
-**          stream.  Used in ResourceReader for corner cases.
-**
-**
-===========================================================*/
-
 using System.Runtime.InteropServices;
 using System.Diagnostics;
 
 namespace System.IO
 {
+    /// <summary>
+    /// Pins a byte[], exposing it as an unmanaged memory stream.  Used in ResourceReader for corner cases.
```

Comment:
cref would not work for byte[] but langword seems to do the trick.
ResourceReader is not imported, so needs full name:

```suggestion
    /// Pins a <see langword="byte[]"/>, exposing it as an unmanaged memory stream.  Used in <see cref="System.Resources.ResourceReader"/> for corner cases.
```